python-package/xgboost/dask.py

```diff
@@ -639,14 +639,15 @@ async def _train_async(client,
 
     workers = list(_get_workers_from_data(dtrain, evals))
     _rabit_args = await _get_rabit_args(len(workers), client)
+    _global_config = config.get_config()
 
     def dispatched_train(worker_addr, rabit_args, dtrain_ref, dtrain_idt, evals_ref):
         '''Perform training on a single worker.  A local function prevents pickling.
 
         '''
         LOGGER.info('Training on %s', str(worker_addr))
         worker = distributed.get_worker()
-        with RabitContext(rabit_args):
+        with RabitContext(rabit_args), config.config_context(**_global_config):
```

Comment:
Does Dask automatically marshal the object `_global_config` to the worker processes? That's pretty cool.

Reply:
Cool most of the time.  Not cool when you start worrying about something got pickled. ;-)